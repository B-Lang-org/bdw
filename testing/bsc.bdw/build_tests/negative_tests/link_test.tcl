
# Compiles the project
handlers::link 

# Creates a new project
handlers::open_project project.bspec 

# Links the project
handlers::link 

# Sets the compilation type
handlers::set_link_type make

# Compiles the project
handlers::link 

# Sets the compilation type
handlers::set_link_type custom_command

# Compiles the project
handlers::link 

# Closes the project 
handlers::close_project

# Exits the worksptation
exit
