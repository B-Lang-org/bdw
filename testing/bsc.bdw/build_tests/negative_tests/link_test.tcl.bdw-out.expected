There is no open project
The current project has not been compiled yet.
 Please compile!
Makefile for the link command is not specified.
The custom command for linking is not specified.
